setup.py

```diff
@@ -14,7 +14,11 @@
 with open('requirements.txt') as f:
     required = f.read().splitlines()
 
-data_files = [("Microsoft.VC90.CRT", glob(r'C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\redist\x86\Microsoft.VC90.CRT\*.*'))]
+import platform
+if platform.system() != "Darwin":
```

Comment:
Is there any reason you chose to specify Darwin and not Windows? It seems that this could create issues on any non-windows platform. Would the following not work for this issue?

```
if platform.system() == "Windows":
    data_files = [("Microsoft.VC90.CRT", glob(r'C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\redist\x86\Microsoft.VC90.CRT\*.*'))]
else:
    data_files = []